breathecode/services/learnpack/actions/open_step.py

```diff
@@ -5,11 +5,12 @@
 
 def open_step(self, webhook):
     # lazyload to fix circular import
-    from breathecode.events.models import Organization
     from breathecode.events.actions import publish_event_from_eventbrite
+    from breathecode.events.models import Organization
 
     org = Organization.objects.filter(id=webhook.organization_id).first()
 
+    # FIXME: undefined variable payload
```

Comment:
Fix it

Reply:
I can't because I don't know which object it's getting thought of parameters

Reply:
@alesanchezr

Reply:
Actually we are not using this function, are we? @alesanchezr